build.sbt

```diff
@@ -147,6 +147,7 @@ lazy val testchipip = (project in file("generators/testchipip"))
 lazy val chipyard = (project in file("generators/chipyard"))
   .dependsOn(testchipip, rocketchip, boom, hwacha, sifive_blocks, sifive_cache, iocell,
     sha3, // On separate line to allow for cleaner tutorial-setup patches
+    mempress,
```

Comment:
Add this after the `constellation` (this breaks some old testing thing if you add it here I think)